cli/src/main/scala/filodb.cli/CsvImportExport.scala

```diff
@@ -68,6 +68,17 @@ trait CsvImportExport {
       case RowSource.AllDone =>
     }
 
+    // There might still be rows left after the latest flush is done, so initiate another flush
```

Comment:
I might add a default explicitly

     private val DefaultTimeout: FiniteDuration = 5 seconds

    protected def parse[T, B](cmd: => Future[T], awaitTimeout: FiniteDuration = DefaultTimeout)(func: T => B): B = 
        func(Await.result(cmd, awaitTimeout))

Why explicitly execute a Future op if the op is run in an explicit Await on the thread?
     protected def actorAsk[B](actor: ActorRef, msg: Any,
                                          awaitTimeout: FiniteDuration = DefaultTimeout)(f: PartialFunction[Any, B]): B = {
           implicit val timeout = Timeout(awaitTimeout)
           parse(actor ? msg, askTimeout)(f)
     }